### DISCUSSION TITLE
Help with vsdiffmerge

### DISCUSSION THREAD

**Discussion by Sandr0oo** [gitextensions/gitextensions#10561] [Q&A] [answered]
How to configure difftool and mergetool? Help me please!
This is how it doesn't work: 
![изображение](https://user-images.githubusercontent.com/61181414/208868332-0e5d17d1-6381-4340-93ac-65859b3ac1b8.png)

**Comment by mstv** [comment]
Works for me.

Does the `vsDiffMerge.exe` really exist in this folder?

![image](https://user-images.githubusercontent.com/36601201/208877133-f32458e1-c162-478a-b1dd-e8d3620ec1e2.png)

What is the output when you run the git command on the command line?
You can copy the arguments from the Git Command Log (hit `F12`).

![image](https://user-images.githubusercontent.com/36601201/208878008-201e38bc-286b-4f0d-98c0-9fde79fd6241.png)

**Reply by Sandr0oo** [reply]
exist:
![firefox_d3bj4bmLYy](https://user-images.githubusercontent.com/61181414/208884310-5ebe79c8-0390-4146-bb7c-4024c6409253.png)
Arguments:
![GitExtensions_WonxN84M64](https://user-images.githubusercontent.com/61181414/208884376-4f1fc86b-419d-41a9-883a-6b69bbb9dfd3.png)
Does not work:
![aBvtHGcDdS](https://user-images.githubusercontent.com/61181414/208885280-f614e7fb-3413-40a3-8c02-cf510bcf147f.png)

**Comment by mstv** [comment, marked as answer]
GE cannot help if the difftool does not support **folder** diffs, as it apparently is the case for vsDiffMerge.
Most likely it silently fails - even without returning a non-zero exit code.

For single files, it works.

![image](https://user-images.githubusercontent.com/36601201/208899755-239490f4-7b76-4cdb-b2f9-c315ef21219a.png)
